openmrs/apps/clinical/formConditions.js

```diff
@@ -31,7 +31,7 @@ Bahmni.ConceptSet.FormConditions.rules = {
 	'Normal nutrition': function(formName, formFieldValues) {
 		var conditions = {show: [], hide: []};
 		var value = formFieldValues['Normal nutrition'];
-		if (value === "No full name") {
+		if (value === "No") {
```

Comment:
@lauravignoli did you mean `value === "No full name" || value === "No"` here as well? (note: the updated skip logic didn't work)